src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+import java.util.Random;
+public class Lottery {
+    public static Ball getRandomBall(){
+        Ball RandomBall = new Ball();
+        RandomBall.setColor(ColorSupplier.getRandomColor());
+        int RandomIndex = new Random().nextInt(101);
```

Comment:
You're creating a new instance of `Random` every time the method is called. It's better to have a single instance as a class-level field to avoid unnecessary object creation.